lib/utils/authorizeSilently.ts

```diff
@@ -29,7 +29,7 @@ export const authorizeSilently = (
 
       cleanup();
 
-      if (event.data && !event.data.error) {
```

Comment:
this wasn't really a valid / useful check before, so writing these tests had at least the value of making me notice it and fix it 🙂